R/pkg/R/DataFrame.R

```diff
@@ -3593,17 +3593,27 @@ setMethod("str",
 #' drop(df, "col1")
 #' drop(df, c("col1", "col2"))
 #' drop(df, df$col1)
+#' drop(df, list(df$col1, df$col2))
 #' }
 #' @note drop since 2.0.0
 setMethod("drop",
           signature(x = "SparkDataFrame"),
           function(x, col) {
```

Comment:
should probably use a different siganture with `...`. Feel free to remove this in this PR, and do it in another PR if you're not used to this.